### ISSUE TITLE
这行代码的名字映射可能存在问题

### ISSUE THREAD

**Issue by Luozeyuisthebest** [Ernest861/NetworkApp#1]
https://github.com/Ernest861/NetworkApp/blob/19c19ab823548ad5fc98ce27215be4d3c8cdf723/app.R#L1393C26-L1393C52
代码审查者您好：
我认为下面这行代码，
"对数似然-G (高斯)" = "loglik-g"
根据后面的代码，可能是     "对数似然-G (高斯)" = "logl-g" 
我并不肯定，这是一种可能性
有人在运行的时候发生了以下问题

![Image](https://github.com/user-attachments/assets/cf68e231-097f-4558-927b-bf5441779c1d)

![Image](https://github.com/user-attachments/assets/a1474f7b-67bf-490a-8beb-b6d712f3e39e)

**Comment by Luozeyuisthebest** [comment]
@Ernest861

**Comment by Luozeyuisthebest** [comment]
我发现我标注的这行代码并不是解决问题的关键。
应该把第utils.R中第1493行代码
    cv_result <- bnlearn::bn.cv(numeric_data, learned_net, loss = "logl-g", k = 10, debug = FALSE)
中的loss = "logl-g"改成loss = "logl"

**Comment by Ernest861** [comment]
你测试过 OK 后可以 push origin 我合并掉
